components/cap-jurisdictions.js

```diff
@@ -0,0 +1,45 @@
+import { LitElement, html, css } from "../lib/lit.js";
+import { fetchJurisdictionsData } from "../lib/data.js";
+
+export default class CapJurisdictions extends LitElement {
+	static properties = {
+		jurisdictionsData: { attribute: false },
+	};
+
+	constructor() {
+		super();
+		this.jurisdictionsData = [];
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		fetchJurisdictionsData(this);
+	}
+
+	render() {
+		return html`
+			<ul>
+				${Object.keys(this.jurisdictionsData)
+					.sort()
+					.map(
+						(jurisdiction) =>
+							html`<li>
+								<h2>${jurisdiction}</h2>
+								<ul>
```

Comment:
This appears to be the short name for the jurisdiction, rather than the full name or the slug. 

The existing case.law website outputs the full name for each jurisdiction, and then appears to use the slug as an anchor link — used by the each link in the sidebar.

![Screenshot 2024-01-11 at 2 03 07 PM](https://github.com/harvard-lil/capstone-static/assets/4039311/7526a5b0-5160-4060-9dcf-17efac078399)

Reply:
Good call. Unfortunately, I don't have that in the data. Another task on our list!

Reply:
Sounds good!

Reply:
Is this already added to the google doc somewhere? Here are the tasks that I see:

- In the data that gets passed to the page, we'll need to render the full name of the legal jurisdiction
- The heading that wraps the full name needs to have an id assigned to it based on the abbreviation for the legal jurisdiction, for example "ar" 
- We'll also need a list that matches the current data structure in `aboutSidebarLinks.js`, which is an array in this shape:
 
```javascript
[
	{
		title: "Arkansas",
		url: "#ark",
	},
]
```

We can use that array to pass to `cap-anchor-list`.  At a glance, I didn't see this in the list — but it sounds like, if I understand correctly, your plan would be to merge this PR in and then create separate tickets to handle this?